support-frontend/assets/components/ticker/contributionTicker.tsx

```diff
@@ -42,9 +42,9 @@ const getInitialTickerValues = (
 		tickerCountType === 'money' ? '/ticker.json' : '/supporters-ticker.json',
 	)
 		.then((resp) => resp.json())
-		.then((data) => {
-			const total = parseInt(data.total, 10);
-			const goal = parseInt(data.goal, 10);
+		.then((data: DataFromServer) => {
+			const total = Math.floor(data.total);
+			const goal = Math.floor(data.goal);
```

Comment:
The types from the server are already `number`s so the call to `parseInt` wasn't actually correct as it expects a `string`.